src/containers/NodesViewer/NodesViewer.js

```diff
@@ -42,60 +62,17 @@ class NodesViewer extends React.PureComponent {
         showControls: true,
     };
 
-    state = {
-        filteredNodes: [],
-        nodesToShow: [],
-    };
-
-    static getDerivedStateFromProps(props, state) {
-        const {nodes, filter} = props;
-        if (!_.isEqual(nodes, state.nodes)) {
-            return {
-                nodes,
-                filteredNodes: NodesViewer.filterNodes(nodes, filter),
-            };
-        }
-        return null;
-    }
-
-    static filterNodes(nodes, filter) {
-        if (filter === ALL) {
-            return nodes;
-        }
-
-        return _.filter(nodes, (node) => {
-            return node.Overall && node.Overall !== 'Green';
-        });
-    }
-
-    static selectNodesToShow(nodes, searchQuery) {
-        let preparedNodes = nodes;
-        if (nodes && Array.isArray(nodes)) {
-            preparedNodes = nodes
-                .map((node) => {
-                    node.uptime = calcUptime(node.StartTime);
-                    return node;
-                })
-                /*  Filter by nodes with the Host field.
-                If a node does not have a Host field it is also
-                included in the filter and displayed with a dash in the corresponding column
-            */
-                .filter((node) => (node.Host ? node.Host.includes(searchQuery) : true));
-        }
-        return preparedNodes;
-    }
-
     onChangeProblemFilter = (filter) => {
```

Comment:
Lets do consistently in this and in the next function: filter, filter or value, value.